### PR TITLE
data: add ams0.json

### PR DESCRIPTION
**Pull request by ams0**
## Changes proposed

Add [ams0](https://github.com/ams0) to EddieHub' LinkFree!

## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

Thank you!

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4386"></a>

### CODE REVIEW

File: data/ams0.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    "Kubernetes",
+    "Service Mesh",
+    "Networking"
+  ],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/bongo" },
+    { "icon": "FaGithub", "url": "https://github.com/ams0" },
+    { "icon": "FaLinkedIn", "url": "https://www.linkedin.com/in/alessandrovozza/" }
+  ],
+  "avatar": "https://github.com/ams0.png",
```
Comment (kumarsonsoff3) [inline data/ams0.json:19]: As of our latest update, we no longer need the `avatar` key. So please remove this from your file. 🛠

```suggestion
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Review by kumarsonsoff3 — DISMISSED** [review]
Looks good, Thanks! 🎉

**Review by rupali-codes — COMMENTED** [review]
lgtm :)

File: data/ams0.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    "Cloud-Native Open Source",
+    "DevOps",
+    "Kubernetes",
+    "Service Mesh",
+    "Networking"
+  ],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/bongo" },
+    { "icon": "FaGithub", "url": "https://github.com/ams0" },
+    { "icon": "FaLinkedIn", "url": "https://www.linkedin.com/in/alessandrovozza/" }
```
Comment (Panquesito7) [inline data/ams0.json:17]: Wrong icon name.
```suggestion
    { "icon": "FaLinkedin", "url": "https://www.linkedin.com/in/alessandrovozza/" }
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by priyankarpal — DISMISSED** [review]
LGTM 👍🏻

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Pull request merged**